types/loadable__server/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for @loadable/server 5.2
+// Type definitions for @loadable/server 5.3
```

Comment:
Could you also change this to the newest version of the library which is 5.8?
https://github.com/smooth-code/loadable-components/releases/tag/v5.8.0

Reply:
Is it `5.8.0` or `5.8`?

Reply:
`5.8` :)